Resources/Prototypes/DeltaV/Body/Parts/feroxi.yml

```diff
@@ -0,0 +1,119 @@
+# TODO: Add descriptions (many)
+# TODO BODY: Part damage
+- type: entity
+  parent: [BaseItem, BasePart]
```

Comment:
all of these need spaces when doing inline lists in yaml like this `[ item1, item2 ]`
this also can just be parented off BasePart and doesnt need BaseItem